easybuild/easyconfigs/d/dorado/dorado-0.8.0-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -76,7 +77,23 @@ _copts = [
     '-DCMAKE_C_FLAGS="$CFLAGS -pthread"',
 ]
 
-configopts = ' '.join(_copts)
+configopts = ' '.join(_copts) + ' '
+
+# disable CMake fiddling with RPATH when EasyBuild is configured to use RPATH linking
+configopts += "$(if %(rpath_enabled)s; then "
+configopts += "echo '-DCMAKE_SKIP_INSTALL_RPATH=YES -DCMAKE_SKIP_RPATH=YES'; fi) "
```

Comment:
Since we have no nicer way to check build_options in easyconfigs (yet?), but this one is a bit cryptic. 

I was thinking at least the config options maybe wouldn't cause any harm to include these when rpath isn't enabled? I wouldn't expect CMAKE_SKIP_RPATH=YES to unwantingly also enable RPATH
maybe fine to just include them in copts?

Reply:
I'm not sure about that...

By default, they're injecting `$ORIGIN` into the RUNPATH section of several libraries they copy, to make sure those libraries can "find" each other.

That's done at install time, after copying library files, so it totally escapes our RPATH compiler/linker wrappers.